Update Terms of Service (#678)
WordPress/openverse-api: api/docs/terms_of_service.md

# Openverse API Terms of Service

> _Effective as of 4 May 2022_

Reviewer: ```suggestion
_Effective as of 5 May 2022_
```

## 1. Your Agreement to the Terms

Thank you for using the Openverse Catalog application programming interface (“Openverse API”).
BY USING THE OPENVERSE API, YOU ACKNOWLEDGE THAT YOU HAVE READ, UNDERSTOOD, AND AGREED TO BE BOUND BY THESE API TERMS OF USE (hereafter, “API Terms”).
By using the Openverse API, you also represent that you have the legal authority to accept the API Terms on behalf of yourself and any party you represent in connection with your use of the Openverse API.
If you do not agree to the API Terms, you are not authorized to use the Openverse API.

## 2. Licensed Uses and Restrictions

In connection with your use of the Openverse API:

  - You must adhere to all rate limits, registration requirements, and comply with all requirements in the Openverse API documentation;

>   - You must not scrape the content in the Openverse Catalog;

Reviewer: Why is this?

Reviewer 2: `- You must not scrape the content in the Openverse Catalog`

I'm also not sure what this line means; is it trying to discourage making a bulk copy of _all_ the data supplied by the API?

Author: I am honestly not sure. The "catalog" itself isn't even public. I'm happy to delete this line 🤷🏼‍♀️

  - You must not use multiple machines to circumvent rate limits or otherwise take measures to bypass our technical or security measures;

>   - You must not operate in a way that negatively affects other users of the API or impedes WordPress's ability to provide its services;

Reviewer: ```suggestion
  - You must not operate in a way that negatively affects other users of the API or impedes the WordPress Foundation's ability to provide its services;
```

Maybe?

Author: Oops, that's much better!

  - You must respect and comply with the terms and conditions applicable to the content and metadata accessible through the Openverse API, including by providing proper attribution to CC-licensed works and respecting the terms and conditions of platforms hosting CC-licensed works.

>   - You must not, and the products you develop must not encourage others to, violate any laws, regulations, or rights of any other person or entity, including intellectual property rights.

Reviewer: ```suggestion
  - You must not, and the products you develop must not, encourage others to, violate any laws, regulations, or rights of any other person or entity, including intellectual property rights.
```

Reviewer 2: ```suggestion
  - You must not, and the products you develop must not, encourage others to violate any laws, regulations, or rights of any other person or entity, including intellectual property rights.
```

😁

Reviewer: Whoops, I think that's where I _meant_ to add the suggested comma but am too sleepy and went a few words too far! :sweat_smile:

## 3. Content and Data Available through the API

The Openverse API aggregates metadata about openly licensed content hosted on websites managed by third parties.
Openverse does not own or control the content or data made available through the API, and does not verify its licensing status or make any representations or warranties about the content or data whatsoever.
You are responsible for independently verifying whether you have the right to use the content and data and what terms and conditions apply to your use.

## 4. Openverse Trademark Use

If you develop an application or other product or service using the Openverse API, you must prominently indicate that it was made using the Openverse API but is not endorsed or certified by Openverse.
You must comply with the [WordPress Foundation Trademark Policy](https://wordpressfoundation.org/trademark-policy/) for all use of the Openverse/WordPress logos and other trademarks.

## 5. Fees

> To help ensure the sustainability of the Openverse project, Openverse reserves the right to charge fees for commercial uses of the Openverse API and/or for heavy usage that places more of a burden on WordPress's systems and technical staff.

Reviewer: ```suggestion
To help ensure the sustainability of the Openverse project, Openverse reserves the right to charge fees for commercial uses of the Openverse API and/or for heavy usage that places more of a burden on the financial supporters of the WordPress Foundation and the maintainers of its systems.
```

WordPress doesn't have any "staff" as far as I know? Just a clarification. I'm sure the legal folks will have a better way of saying this.

## 6. Changes to the Terms

Openverse is actively developing its product vision and strategy and may limit access to, and/or change the terms that apply to, the Openverse API at any time as plans take shape.
If we make changes that we feel are material, we will make reasonable efforts to update developers who we know are actively using the Openverse API.

> Your continued use of the Openverse API after new and/or revised API Terms are effective indicates that you have read, understood, and agreed to those terms.

Reviewer: ```suggestion
Your continued use of the Openverse API after new and/or revised API Terms are effective indications that you have read, understood, and agreed to those terms.
```
